### PR TITLE
New rule doc pages

### PR DESCRIPTION
**Pull request by tvsonar**

### CODE REVIEW

File: src/CSharp.CodeAnalysis.DocGenerator/packages.config [hunk 1]
```diff
@@ -4,6 +4,7 @@
   <package id="Microsoft.CodeAnalysis.Analyzers" version="1.0.0-rc2" targetFramework="net452" userInstalled="true" />
   <package id="Microsoft.CodeAnalysis.Common" version="1.0.0-rc2" targetFramework="net452" userInstalled="true" />
   <package id="Microsoft.CodeAnalysis.CSharp" version="1.0.0-rc2" targetFramework="net45" userInstalled="true" />
+  <package id="Newtonsoft.Json" version="7.0.1" targetFramework="net45" userInstalled="true" />
   <package id="RazorEngine" version="3.7.0" targetFramework="net45" userInstalled="true" />
```
Comment (dbolkensteyn) [inline src/CSharp.CodeAnalysis.DocGenerator/packages.config]: @tvsonar can we drop this dependency now?

Comment (dbolkensteyn) [inline src/CSharp.CodeAnalysis.DocGenerator/packages.config]: same for ASP.NET Razor

Comment (tvsonar) [inline src/CSharp.CodeAnalysis.DocGenerator/packages.config]: @dbolkensteyn good catch, thanks. It's dropped now.

**Comment by dbolkensteyn** [comment]
LGTM

**Pull request merged**